### PR TITLE
Clarify telemetry documentation

### PR DESCRIPTION
**Pull request by amytai**
- Update the name of the API function to get telemetry JSON
- Clarify distinction between per-database and cluster-wide telemetry
- General nits

### CODE REVIEW

File: using-timescaledb/telemetry.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # Telemetry and Version Checking
-Starting in v0.12.0 we enable anonymous usage sharing to help us better understand and assist Timescale users. We emphasize that privacy of our users is paramount, so we do not collect any identifying information. The following is an example of the JSON that is sent to our servers about a specific deployment:
+Starting in v0.12.0 we enable anonymous usage sharing to help us better understand and assist Timescale users, as
```
Comment (dianasaur323) [inline using-timescaledb/telemetry.md]: We might need to change this to a different version depending on when and what we release

**Comment by dianasaur323** [comment]
Ok, looks good to me, other than that little note I put in that we should fix before we make this live

File: using-timescaledb/telemetry.md [hunk 2]
```diff
@@ -39,32 +44,38 @@ Although we invite our community to help us keep improving our
 product, we do understand when users would like to disable telemetry. Note that
 disabling telemetry also disables the version checking functionality.
 
-To turn off telemetry, simply include the following line in your `postgresql.conf` file:
+Telemetry is sent on a per-database basis, so users can disable telemetry for specific databases or for an entire
+cluster.
+
+To turn off telemetry for an entire cluster, simply include the following line in your `postgresql.conf` file:
```
[the remark this answers is not in the document]
Comment (mfreed) [reply using-timescaledb/telemetry.md]: Agree with Solar.  Postgres term is "cluster", but this is confusing for non-expert users.  Thus I believe we more commonly use "instance"

[the remark this answers is not in the document]
Comment (mfreed) [reply using-timescaledb/telemetry.md]: Agree with Solar. Postgres term is "cluster", but this is confusing for non-expert users. Thus I believe we more commonly use "instance"

[the remark this answers is not in the document]
Comment (amytai) [reply using-timescaledb/telemetry.md]: fixed

**Pull request merged**